pkg/staticnetworkconfig/generator_test.go

```diff
@@ -20,7 +20,9 @@ func TestStaticNetworkConfig(t *testing.T) {
 var _ = Describe("StaticNetworkConfig", func() {
 	var (
 		staticNetworkGenerator = StaticNetworkConfigGenerator{log: logrus.New()}
+		mockNMState            = MockNmstate{}
```

Comment:
Seems strange that we're not actually using this anywhere in these tests.

Given the instability we've seen in nmstate wouldn't it be good to not mock this and actually validate that it's working correctly in these tests?

Maybe we have something like that in subsystem already?